backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/session/UserSessionRestController.java

```diff
@@ -58,7 +60,8 @@ public JSONLookupValue setLanguage(@RequestBody final JSONLookupValue value)
 	{
 		final String adLanguage = value.getKey();
 		userSessionRepo.setAD_Language(userSession.getLoggedUserId(), adLanguage);
-		
+		documentCollection.cacheReset(false);
```

Comment:
why `forgetNotSavedDocuments==false`?

Reply:
I saw it as the preferred way in the code-base:
- `de.metas.ui.web.dataentry.interceptor.DataEntryInterceptorUtil#resetCacheFor(de.metas.dataentry.model.I_DataEntry_Tab)` always calls it with `false`
- `de.metas.ui.web.debug.DebugRestController#cacheReset` the default value for `forgetNotSavedDocuments` is `false`

In this case I also didn't think I'm doing something to warrant using `forgetNotSavedDocuments=true`

Reply:
agree to go with forgetNotSavedDocuments=false